cli/internal/git/pull.go

```diff
@@ -29,19 +36,26 @@ func pull(gitUrl string, targetFolder string) {
 
 	gitCred := FindAuthForHost(gitUrl)
 
+	// This is here for backwards compatibility
```

Comment:
Can you clarify this comment?  I think I understand what you are saying in that we now support with and with the tag, but to be clear the default and normal behavior we will document/encourage will use the tag as that will be very important for upcoming diff engine work

Reply:
Good catch! That comment was placed when I did the first commit to switch to full mirrors and although I have changed the variable below it since, I missed deleting the comment. This comment needs to be deleted. I can push a deletion commit now, or include one when I force push a squash. I've just been keeping everything as separate commits lately so reviewing changes such as tag branch name targets could be done in steps, which I assume is how it has been missed until now.

Reply:
either way we'll just need all the commits in before final test/review, up to you, thanks.